pkg/config/validation.go

```diff
@@ -174,3 +176,27 @@ func validatePodIntegrationOptions(c *configapi.Configuration) field.ErrorList {
 
 	return allErrs
 }
+
+func validateFairSharing(c *configapi.Configuration) field.ErrorList {
+	fs := c.FairSharing
+	if fs == nil {
+		return nil
+	}
+	var allErrs field.ErrorList
+	if !fs.Enable && len(fs.PreemptionStrategies) != 0 {
+		allErrs = append(allErrs, field.Invalid(fsPreemptionStrategiesPath, fs.PreemptionStrategies, "Must be empty when fair sharing is disabled"))
+	}
+	strategies := sets.New[configapi.PreemptionStrategy]()
+	validStrategies := []configapi.PreemptionStrategy{configapi.LessThanInitialShare, configapi.LessThanOrEqualToFinalShare}
```

Comment:
I think we should either validate against non-supported configurations or support them. I'm leaning towards validation, because it will keep the code simpler. IIUC this is the only non-supported option for now: `[LessThanInitialShare, LessThanOrEqualToFinalShare]` .

Reply:
Changed validation to have an explicit list of supported combinations.